packages/rs-dpp/src/errors/consensus/consensus_error.rs

```diff
@@ -28,6 +28,9 @@ pub enum ConsensusError {
     #[error(transparent)]
     BasicError(BasicError),
 
+    #[error("system error: {0}")]
```

Comment:
Why it's protocol consensus? I guess we just need to send a protocol error with ABCI response code. All codes below 1000 meant to be from DAPI and Drive errors (not DPP, protocol). Internal ABCI error code is 1.

Reply:
@shumkov so what do you want to do?